README.md

```diff
@@ -117,16 +85,16 @@ Latest database dump is available at http://acousticbrainz.org/download. You
 need to download full database dump from this page and use it during database
 initialization:
 
-    $ python manage.py init_db path_to_the_archive
+    $ docker-compose -f docker/docker-compose.dev.yml run webserver python2 manage.py init_db path_to_the_archive
```

Comment:
this will need volumes if the path isn't in the current directory and we're not mounting it as a volume